docs/docsite/rst/contributor/communication.rst

```diff
@@ -0,0 +1,39 @@
+.. _communication:
+
+Communication
+=============
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+.. _code_of_conduct:
+
+Code of Conduct
+---------------
+
+All communication and interactions in the Ansible Community are governed by our :ref:`code_of_conduct`. Please read and understand it!
+Reach out to our community team at `codeofconduct@ansible.com <mailto:codeofconduct@ansible.com>`_ if you have any questions or need assistance.
+
+.. _forum:
+
+Forum
+-----
+
+Join the `Ansible Forum <https://forum.ansible.com>`_ as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `awx`, `ee`, and  `documentation`.
+* `Posts tagged with 'awx' <https://forum.ansible.com/tag/awx>`_: subscribe to participate in project/technology-related conversations. There are other related tags in the forum you can use.
+* `Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_: used to announce releases and important changes.
+* `Social Spaces <https://forum.ansible.com/c/chat/4>`_: gather and interact with fellow enthusiasts.
+* `News & Announcements <https://forum.ansible.com/c/news/5>`_: track project-wide announcements including social events.
+
+For more information on the forum navigation, see `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ post.
+
+Matrix
+------
+
+For real-time interactions, conversations in the Ansible AWX community happen over the Matrix protocol in the following channels:
```

Comment:
```suggestion
For real-time interactions, conversations in the AWX community happen over the Matrix protocol in the following channels:
```

Reply:
Somewhere there is a guideline about it always being "AWX" and never "Ansible AWX". I got caught out by this myself.